lib/fileHandler.js

```diff
@@ -7,12 +7,16 @@ function readInput(filePath) {
   const absolutePath = getAbsolutePath(filePath);
   const extension = extname(absolutePath);
   const data = readFileSync(absolutePath);
-  return extension === '.ndjson' ? parseNdjson(data) : parseJson(data);
+  return extension === '.ndjson' ? parseNdjson(data) : JSON.parse(data);
```

Comment:
Since the usage of  `parseJson` was removed here, is there still a need to use it in `parseNdjson`?